src/libostree/ostree-repo-pull.c

```diff
@@ -1819,7 +1819,7 @@ scan_commit_object (OtPullData                 *pull_data,
   if (pull_data->timestamp_check)
     {
       /* We don't support timestamp checking while recursing right now */
-      g_assert (ref);
+      g_assert (ref != NULL);
```

Comment:
Did you run this through Coverity again or is it a guess?

Reply:
It's a guess. I'm not sure how to request another run. Will look into it.

Reply:
It didn't make a difference, so I dropped that hunk. I left the other two though since I think they're still useful and reworked the commit message. :arrow_down:

Reply:
(To clarify, the first one is definitely a good check to have, while the second one is just Coverity not being smart enough. Though they both did squash warnings).